content/id/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Memperluas Kubernetes Cluster Anda
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Kubernetes sangat mudah dikonfigurasi dan dikembangkan. Sehingga,
+jarang membutuhkan fork atau menambahkan patches ke kode projek Kubernetes.
+
+Panduan ini menjelaskan pilihan untuk menyesuaikan Kubernetes cluster.
+Ini ditujukan kepada {{< glossary_tooltip text="cluster operators" term_id="cluster-operator" >}} yang ingin
+memahami bagaimana menyesuaikan kluster Kubernetes dengan kebutuhan lingkungan kerja mereka.
+Pengembang yang prospektif {{< glossary_tooltip text="Platform Developers" term_id="platform-developer" >}} atau {{< glossary_tooltip text="Contributors" term_id="contributor" >}} Projek Kubernetes juga mendapatkan manfaat dari
+dokumen ini sebagai pengantar apa saja poin dan pola yang ada, untung-rugi dan batasannya.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Gambaran
```

Comment:
```suggestion
## Ikhtisar
```

To make our translation documents uniform, we use "Ikhtisar"

Reply:
done